packages/scene-composer/src/components/three-fiber/AnimationComponent/AnimationComponent.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { Fragment } from 'react';
+import * as THREE from 'three';
+import {  useFrame } from '@react-three/fiber';
+import { GLTF } from 'three/examples/jsm/loaders/GLTFLoader';
+import { useGLTF } from '../ModelRefComponent/GLTFLoader';
+import { useSceneComposerId } from '../../../common/sceneComposerIdContext';
+import { ISceneNodeInternal, useEditorState, useViewOptionState } from '../../../store';
+
+import { IAnimationComponent } from '../../../interfaces';
+
+function pauseAnimation(gltfModel, AnimationMixer, isPaused, AnimationList){
+  try{
+    AnimationList.forEach( function ( clip ) {
+      const animation = THREE.AnimationClip.findByName( gltfModel, clip );
+      AnimationMixer.clipAction(animation).paused = isPaused
```

Comment:
do this function both pause and unpause? from the function name I'd expect only unpause. maybe add a function comment if it does more then that.